src/poetry/console/commands/init.py

```diff
@@ -142,7 +142,7 @@ def handle(self) -> int:
         question = self.create_question(
             f"Author [<comment>{author}</comment>, n to skip]: ", default=author
         )
-        question.set_validator(lambda v: self._validate_author(v, author))
+        question.set_validator(lambda v: InitCommand._validate_author(v, author))
```

Comment:
No need to change this line. (We can access a static method via `self`.)

Reply:
TIL